server/http.go

```diff
@@ -298,3 +234,43 @@ func splitInstancePath(route string) (instanceURL string, remainingPath string)
 	}
 	return string(id), leadingSlash + strings.Join(ss[2:], "/")
 }
+
+func (p *Plugin) handleResponse(fn func(w http.ResponseWriter, r *http.Request) (int, error)) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		status, err := fn(w, r)
+		switch {
+		case err == nil && status == http.StatusOK:
+			p.API.LogDebug("OK: ", "Status", strconv.Itoa(status), "Path", r.URL.Path, "Method", r.Method, "query", r.URL.Query().Encode())
+		case status == 0:
+			p.API.LogDebug("Passed to another router: ", "Path", r.URL.Path, "Method", r.Method)
+		case err != nil:
+			p.API.LogError("ERROR: ", "Status", strconv.Itoa(status), "Error", err.Error(), "Path", r.URL.Path, "Method", r.Method, "query", r.URL.Query().Encode())
```

Comment:
Let's use `LogWarn` for this instead

```suggestion
			p.API.LogWarn("ERROR: ", "Status", strconv.Itoa(status), "Error", err.Error(), "Path", r.URL.Path, "Method", r.Method, "query", r.URL.Query().Encode())
```

Reply:
Thanks, updated the log level.